### PR TITLE
Added weighted BCE

### PR DESCRIPTION
**Pull request by EvgeniyS99**

### CODE REVIEW

**Review by AlexeyKozhevin — CHANGES_REQUESTED** [review]

File: batchflow/models/torch/losses/binary.py [hunk 1]
```diff
@@ -191,3 +193,27 @@ def forward(self, prediction, target):
...
+        mask[mask == 0] = num_positive / (num_positive + num_negative)
+
+        loss = torch.zeros(1, device=prediction.device)
+        loss += F.binary_cross_entropy_with_logits(prediction, target, weight=mask)
+
+        return loss
+
+losses = [WeightedBCE(), Dice()]
+weights = [.5, .5]
+comboloss = Weighted(losses, weights)
```
Comment (AlexeyKozhevin) [inline batchflow/models/torch/losses/binary.py:219]: Do you really need it here? There is not a problem to make it in your notebook:

ComboLoss = Weighted([WeightedBCE(), Dice()], [.5, .5])

Nevertheless, `comboloss` here is inappropriate naming.

**Review by AlexeyKozhevin — CHANGES_REQUESTED** [review]

File: batchflow/models/torch/losses/binary.py (hunk 1 above)
Comment (EvgeniyS99) [reply batchflow/models/torch/losses/binary.py:219]: I think, ye, but I thought I should present the full loss used in my models

Comment (EvgeniyS99) [reply batchflow/models/torch/losses/binary.py:219]: > Do you really need it here? There is not a problem to make it in your notebook:
> 
> ComboLoss = Weighted([WeightedBCE(), Dice()], [.5, .5])
> 
> Nevertheless, `comboloss` here is inappropriate naming.

Why is comboloss inappropriate naming? The bce loss is **combined** with dice loss --> **comboloss**

File: batchflow/models/torch/losses/binary.py [hunk 2]
```diff
@@ -191,3 +191,23 @@ def forward(self, prediction, target):
         sensitivity = (squared_error * inverse).sum() / (inverse.sum() + self.eps)
 
         return self.r * specificity + (1 - self.r) * sensitivity
+
+
+
+class BalancedWeightedBCE(nn.Module):
```
Comment (SergeyTsimfer) [inline batchflow/models/torch/losses/binary.py:197]: `CrossEntropyLoss` from `batchflow.models.torch.losses.core.py` already can do all of that. Maybe, make inherit BCE / make it a special case of it somehow?
